src/hugr/views/sibling.rs

```diff
@@ -0,0 +1,584 @@
+//! Views for HUGR sibling subgraphs.
+//!
+//! Views into subgraphs of HUGRs within a single level of the
+//! hierarchy, i.e. within a sibling graph. Such a subgraph is
+//! represented by a parent node, of which all nodes in the sibling subgraph are
```

Comment:
nit: this `represented by` isn't true anymore - "can be created from" if you want, but probably just represented by or *induced* by a subset of nodes

Reply:
Oh good spot! I missed this one when I rewrote the code.